herbert_core/configuration/@config_store/config_store.m

```diff
@@ -34,6 +34,12 @@
             if is_virtual
                 newStore.config_folder_name_ = ['mprogs_config_',type];
             end
+            [is_virtual,build_name] = is_jenkins();
+            if is_virtual
+                [~,build_name] = fileparts(build_name); % remove all possible folder path's of the build name 
```

Comment:
this should in `tmpdir` as we're already managing `TMP` environment variable to create job-unique temp folder locations.

Reply:
This one defines folder name. There is function called after that, which defines the folder location. 
I do not want the logic of the program running on Jenkins to be different from the one, running locally.  If we want folder to be in tmp and bring this logic into config_manager, the `create_config_folder` function should be changed. 
Currently we setting up tmp_dir in test configuration. This is fine -- `create_config_folder` picks up this. (or should pick it up)